docs/proxy/datacenter_proxy/examples.md

```diff
@@ -0,0 +1,541 @@
+---
+title: Examples
+description: Learn how to connect to Apify's datacenter proxies from your application with Node.js (axios and got), Python 2 and 3 and PHP using code examples.
+paths:
+    - proxy/datacenter-proxy/nodejs-examples
+    - proxy/datacenter-proxy/python-examples
+    - proxy/datacenter-proxy/php-examples
+    - proxy/datacenter-proxy/examples
+---
+<!-- Watch out! This file is hard to read because our own "marked-tabs" language fences aren"t syntax-highlighted -->
+
+# Connect to datacenter proxies
+
+This page contains code examples for connecting to [datacenter proxies]({{@link proxy/datacenter_proxy.md}}) using [Apify Proxy](https://apify.com/proxy).
+
+See the [connection settings]({{@link proxy/connection_settings.md}}) page for connection parameters.
+
+## [](#using-the-apify-sdk) Using the Apify SDK
+
+If you are developing your own Apify [actor]({{@link actors.md}}) using the [Apify SDK](https://sdk.apify.com), you can use Apify Proxy in:
+
+* [PuppeteerCrawler](https://sdk.apify.com/docs/api/puppeteer-crawler#docsNav) using the [createProxyConfiguration()](https://sdk.apify.com/docs/api/apify#apifycreateproxyconfigurationproxyconfigurationoptions) function.
+* [CheerioCrawler](https://sdk.apify.com/docs/api/cheerio-crawler#docsNav) using the [createProxyConfiguration()](https://sdk.apify.com/docs/api/apify#apifycreateproxyconfigurationproxyconfigurationoptions) function.
+* [requestAsBrowser()](https://sdk.apify.com/docs/api/utils#utilsrequestasbrowseroptions) function by specifying proxy configuration in the options.
+* [launchPuppeteer()](https://sdk.apify.com/docs/typedefs/launch-puppeteer#docsNav) by specifying the configuration in the function's options.
+
+The Apify SDK's [ProxyConfiguration](https://sdk.apify.com/docs/api/proxy-configuration) enables you to choose which proxies you use for all connections. You can inspect the current proxy's URL and other attributes using the [ProxyInfo](https://sdk.apify.com/docs/typedefs/proxy-info) property in your crawler's [page function](https://sdk.apify.com/docs/typedefs/cheerio-crawler-options#handlepagefunction).
+
+### [](#rotate-ip-addresses) Rotate IP addresses
+
+IP addresses for each request are selected at random from all available proxy servers.
+
+To use IP addresses from specific proxy groups, add a `groups` [property](https://sdk.apify.com/docs/api/proxy-configuration#docsNav)
+to `createProxyConfiguration()` and specify the group names. For example, `groups: ["GROUP_NAME_1", "GROUP_NAME_2"]`.
+
+```marked-tabs
```

Comment:
The examples here all use auto proxy, which in this case since proxy configuration is used makes it hard to imagine how to actualy provide the groups mentioned above the example. Maybe also provide example of providing a proxy group?

Reply:
I added an example in the SDK section